demisto_sdk/commands/common/tools.py

```diff
@@ -818,7 +797,7 @@ def find_type(path: str = '', _dict=None, file_type: Optional[str] = None, ignor
         if 'preProcessingScript' in _dict:
             return FileType.INCIDENT_TYPE
 
-        if 'regex' in _dict or checked_type(path, JSON_ALL_INDICATOR_TYPES_REGEXES):
+        if 'regex' in _dict or 'reputations' in _dict:
```

Comment:
Correct me if I'm wrong but `reputations` field is used to locate "reputations.json" right?
If so add a comment about this so devs won't wonder why there isn't a `reputations` field in the schema

also please run validate command on `Packs/CommonTypes/IndicatorTypes` to make sure this works as intended.

Reply:
yes, exactly.
I ran validate on all the repo already to check.
but now ran what you asked too
![image](https://user-images.githubusercontent.com/58938354/90782583-b211ab00-e307-11ea-8e77-615b474bfc79.png)

Reply:
@orhovy - so please add the comment then and this is great

Reply:
I did